kas-poky-rpi.yml

```diff
@@ -0,0 +1,58 @@
+header:
+  version: 8
+
+machine: raspberrypi4
+distro: poky
+target:
+  - core-image-base
+
+repos:
+  meta-raspberry:
+
+  poky:
+    url: https://git.yoctoproject.org/git/poky
```

Comment:
It would be preferred if we can use oe-core/bitbake repos here.

Reply:
@kraj In fact, I tried to create a kas file that follows the dependency recommendations in the README file (to facilitate the build configuration for the end user) with poky and so on. 

And as suggested by @pbrkr , I renamed this one kas-poky*.

So, maybe we can add another kas file for a "no-distro" configuration with the oe-core/bitbake repos ?

Reply:
thats fine, I see that we ask for poky too but to keep in sync with readme nodistro is important too

Reply:
Sure @kraj , I will add a kas file for nodistro in another PR ;)